dbt/adapters/greenplum/impl.py

```diff
@@ -1,135 +1,30 @@
-from datetime import datetime
-from dataclasses import dataclass
 from typing import Optional, Set, List, Any
 from dbt.adapters.base.meta import available
-from dbt.adapters.base.impl import AdapterConfig
-from dbt.adapters.sql import SQLAdapter
 from dbt.adapters.greenplum import GreenplumConnectionManager
 from dbt.adapters.greenplum.relation import GreenplumColumn
 from dbt.adapters.greenplum.relation import GreenplumRelation
-from dbt.dataclass_schema import dbtClassMixin, ValidationError
-import dbt.exceptions
-import dbt.utils
-
+from dbt.adapters.postgres.impl import PostgresIndexConfig, PostgresConfig, PostgresAdapter
+from dbt.adapters.postgres import impl
 
 # note that this isn't an adapter macro, so just a single underscore
-GET_RELATIONS_MACRO_NAME = "greenplum_get_relations"
-
+impl.GET_RELATIONS_MACRO_NAME = "greenplum_get_relations"
```

Comment:
I'm not sure about this decision

Reply:
Unfortunately, I got following error
`dbt could not find a macro with the name "postgres_get_relations" in any package`

Reply:
ok so I see now. for this one you may have to copy `postgres_get_relations` and rename it to `greenplum_get_relations` because it isn't double-underscored. i'm going to look into why that is the case, because it shouldn't be! perhaps we can fix this in the upstream dbt-postgres repo

Reply:
just opened https://github.com/dbt-labs/dbt-core/issues/5301 to find out more!

Reply:
I decided to remove `greenplum_get_relations`, because it is the same macro as `postgres_get_relations`.
If it changes to double-underscored, I will fix it.